tests/MongoDB.Driver.Tests/Specifications/read-write-concern/OperationTestRunner.cs

```diff
@@ -0,0 +1,108 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
+using MongoDB.Driver.Tests.Specifications.Runner;
+using Xunit;
+
+namespace MongoDB.Driver.Tests.Specifications.read_write_concern
+{
+    public class OperationTestRunner : MongoClientJsonDrivenTestRunnerBase
+    {
+        private HashSet<string> _defaultCommandsToNotCapture;
+
+        protected override HashSet<string> DefaultCommandsToNotCapture
+        {
+            get
+            {
+                if (_defaultCommandsToNotCapture == null)
+                {
+                    var defaultCommandsToNotCapture = base.DefaultCommandsToNotCapture;
+                    defaultCommandsToNotCapture.Add("find");
+                    _defaultCommandsToNotCapture = defaultCommandsToNotCapture;
+                }
+
+                return _defaultCommandsToNotCapture;
+            }
+        }
+
+        protected override string[] ExpectedTestColumns => new[] { "description", "operations", "outcome", "expectations", "async" };
+
+        // public methods
+        [SkippableTheory]
+        [ClassData(typeof(TestCaseFactory))]
+        public void Run(JsonDrivenTestCase testCase)
+        {
+            SetupAndRunTest(testCase);
+        }
+
+        protected override void VerifyCollectionOutcome(BsonDocument outcome, string database, string inputCollectionName)
```

Comment:
This whole method should be in the base class.

Reply:
done